### ISSUE TITLE
Understand what type of PRA clearance requirements is needed in one time and ongoing pac activities

### ISSUE THREAD

**Issue by widal001** [HHS/simpler-grants-gov#1419]
### Summary

TODO add summary

### Acceptance criteria

- [ ] TODO add acceptance criteria

**Comment by rishalee** [comment]
-Posted in internal-grants slack on 4/9

**Comment by rishalee** [comment]
@margaretspring @andycochran we ok with closing this out? https://betagrantsgov.slack.com/archives/C05T74HRWRL/p1712688306403599

**Comment by andycochran** [comment]
Yeah. I think the answer in that thread resolves this. We might document it if there's somewhere in the Co-design Group planning materials you think appropriate
